cmd/telliot/main.go

```diff
@@ -4,15 +4,12 @@
 package main
 
 import (
-	"fmt"
-
 	"github.com/alecthomas/kong"
 	"github.com/tellor-io/telliot/pkg/cli"
 )
 
 func main() {
 	//lint:ignore faillint it should print to console
-	fmt.Printf(cli.VersionMessage, cli.GitTag, cli.GitHash)
```

Comment:
We still want to show this, maybe as a log, but add it somewhere that will not be printed in the help doc.